napari/_vispy/__init__.py

```diff
@@ -1,3 +1,17 @@
+import logging
+
+import qtpy
+from vispy import app
+
+# set vispy application to the appropriate qt backend
```

Comment:
honestly, you can remove the whole `from vispy import app` and it's still fine.  I just left it in there cause it was in `napari.__init__`.  But vispy [tries these core backends](https://github.com/vispy/vispy/blob/7f3f6f60c8462bb8a3a8fa03344a2e6990b86eb2/vispy/app/backends/__init__.py#L15-L27) (defaulting to the Qt backends first).  So the result is the same either way.  It's conceivable that there is some combination of environment that might mess this up (but I think it would also be messed in master)

Reply:
Interesting, I think we might have needed it a long time ago, but if we don't now could drop, or could just leave in to be explicit. I don't mind, but am fine with it staying. Your call if you want to drop

Reply:
eh, I don't mind it there.  It reminds us that `vispy.app.use_app` exists.